DNN Platform/Website/Templates/Default Website.template

```diff
@@ -2814,93 +2814,6 @@
         </pane>
       </panes>
     </tab>
-    <tab sku="EE,PE,CE">
-      <containersrc />
-      <description>[Tab.AdvancedSettings.Description.Text]</description>
```

Comment:
Should we may remove the Description and Title keys from the template resource file.